src/framework/ui/topNavigationBar/topNavigationBarAction.component.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
In terms of testing light-weight subcomponents prefer creating one spec file for whole component which will describe tests for component and it's children. This will help you avoid creation multiple spec configuration and snapshot files.
Consider refactoring

Reply:
Done.